homework.js

```diff
@@ -0,0 +1,97 @@
+// 1
+const summation = function (num) {
+    let sum;
+    for (let i = 0; i <= num; i++) {
+        sum += i;
+    }
+    return sum;
+}
+
+// 2
+const summation = function (num) {
+    let sum;
+    for (let i = 0; i <= num; i++) {
+        if (i % 2 === 0) {
+            sum += i;
+        }
+
+    }
+    return sum;
+}
+
+// 3
+const avg = function (numsArray) {
+    let total = 0;
+    for (let i = 0; i < numsArray.length; i++) {
+        total += numsArray[i];
+    }
+    return total / numsArray.length;
+}
+
+// 4
+const reverse = function (word) {
+    let lettersArray = word.split('');
+    return lettersArray.reverse().join('');
+}
+
+// 5
+const addDashes = function (wordsArray) {
+    return wordsArray.join('-');
+}
+
+// 6
+const countUpAndDown = function (num) { 
+    let result = []
+    for (let i = 1; i <= num; i++) {
+        result.push(i);
+    }
+    for (let i = num; i >= 1; i--) {
+        result.push(i);
+    }
+    return result.join(" ");
```

Comment:
the number in the middle shouldn't repeat. countUpAndDown(3) should return "1 2 3 2 1", not "1 2 3 3 2 1".